app/javascript/application.js

```diff
@@ -0,0 +1,14 @@
+// Entry point for the build script in your package.json
+import '@hotwired/turbo-rails';
+import './controllers';
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+
+ReactDOM.render(
+  <React.StrictMode>
```

Comment:
- [ ] Also do ensure that you set up redux in your app that has a store, an action, and a reducer that uses the URL to the endpoint from the Rails API.